### ISSUE TITLE
On IDE 1.6.7: Error with uploading

### ISSUE THREAD

**Issue by nouser2013** [esp8266/arduino-esp8266fs-plugin#7]
Greetings,

I'm on IDE 1.6.7 and it fails with the upload. I set everything up as described, and I can upload a sketch with no problems, and it seems to compile the 3 MB of file system binary, but uploading results in these messages:

```
[SPIFFS] data   : U:\ ..... \main\data
[SPIFFS] size   : 3052
[SPIFFS] page   : 256
[SPIFFS] block  : 8192
/test.txt

[SPIFFS] upload : C:\ [...] 8b9a9.spiffs/main.spiffs.bin
[SPIFFS] reset  : ck
[SPIFFS] port   : COM8
[SPIFFS] speed  : 115200
[SPIFFS] address: 0x100000

warning: espcomm_sync failed
error: espcomm_open failed
SPIFFS Upload failed!
```

Am I doing something wrong?

Thanks

**Comment by jongross** [comment]
I am also having this error with Arduino 1.6.7 - any ideas on where to start?

SPIFFS] data   : [...]/microprocessor/esp8266/data
[SPIFFS] size   : 3052
[SPIFFS] page   : 256
[SPIFFS] block  : 8192
/configuration.json
[SPIFFS] upload : /var/folders/b9/qxp1bgjn2ylg5q4xryzs5j6c0000gn/T/build711ec71dfd91376c16d149a0d722a494.spiffs/esp8266.spiffs.bin
[SPIFFS] address: 0x100000
[SPIFFS] reset  : ck
[SPIFFS] port   : /dev/cu.SLAB_USBtoUART
[SPIFFS] speed  : 115200

warning: espcomm_sync failed
error: espcomm_open failed
SPIFFS Upload failed!

**Comment by me-no-dev** [comment]
looks like esptool problem. You do have the serial monitor closed right? You can not have it open and upload like it's done with the sketches, because we do not have access from the tool to the serial port to disconnect it.

**Comment by jongross** [comment]
yeah it was closed - I just figured out the issue - the connection I have here to the chip is supposed to trigger bootload mode before doing anything but it isn't happening, so ground pin0 and rebooting the chip worked.

Thanks for the quick response.

**Comment by me-no-dev** [comment]
BTW the new version of the plugin uploads over OTA as well :) you might find it easier and much faster

**Comment by jongross** [comment]
I'm implementing that piece now, cheers!

**Comment by kdupke** [comment]
I missed to bring my ESP-01s into flash mode before using the tool. Please add this to the documentation.
